doc/developers/auth/cert.rst

```diff
@@ -1,8 +1,12 @@
 Certificate Authentication
 ==========================
 
-CCF's primary authentication mechanism relies on users specifying their certificate and private key in the TLS handshake (e.g. using the ``--key`` and ``--cert`` argument to ``curl``).
+User identities in CCF are X.509 certificates. They can be added or removed via governance proposals, which are subject to the consortium constitution rules (see :ref:`members/open_network:Adding Users`).
 
-Users should have previously been trusted by the consortium of members for them to use this authentication mechanism (see :ref:`members/open_network.html:Adding Users`). 
+Requests emitted by users can be authenticated one of two ways:
```

Comment:
minor: consider changing emitted to sent, i think this is more precise

Reply:
Done